book/src/bridge-module/functions/README.md

```diff
@@ -141,7 +141,8 @@ mod ffi {
 	}
 
     extern "Rust" {
-        #[swift_bridge(into_return_type)]
+        // To make `into_return_type` more consistent wiht other attributes, we use 'return_with'
```

Comment:
We don't need this comment about `into_return_type` since it will no longer exists after this commit. 

You can delete it everywhere that you introduced it.